.github/workflows/tests-nightly.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     strategy:
       matrix:
         testOptions: [ "-PtestTag=unit", "-PtestTag=integration", "-PtestTag=integration -PtestInstallFromGitHub" ]
+        testServer: ["", "-Pserver=terra-verily-autopush"]
```

Comment:
Can you change the default option to specify the server also: "-Pserver=terra-dev". I think that makes it clearer what the two options are.